Dockerfile

```diff
@@ -80,7 +80,7 @@ RUN set -ex && cd ~ \
 
 # install awscliv2, disable default pager (less)
 ENV AWS_PAGER=""
-ARG AWSCLI_VERSION=2.0.9
+ARG AWSCLI_VERSION=2.0.10
```

Comment:
Am I looking in the wrong place to see the difference?
https://github.com/aws/aws-cli/blob/develop/CHANGELOG.rst

How did you know to update this?

Reply:
I use homebrew to track the updates, but that has releases for 1.x and 2.x versions of the cli. 

https://github.com/aws/aws-cli/blob/v2/CHANGELOG.rst has the v2 CHANGELOG